### ISSUE TITLE
Search for room feature

### ISSUE THREAD

**Issue by peeroman** [neokoenig/RoomBooking#30]
Hi, 

I would love to see a feature for searching for room.
Would be good also to put rooms in the buliding/floor structure.

Searching for room should be possible by attributes:
- multimedia (e.g. projector=yes)
- capacity (e.g. 4 persons)
- avaliablity (e.g. from: 8:00 to: 9:00 for tomorrow)

It is essential when dooing bookings.

**Comment by kcits** [comment]
I am curious what you mean by searching by room.  Are you wanting to search the rooms by resources, capacity and availability?

Resources (your multimedia example) would actually need to search available Resources not rooms.

I have added to our install: Max Capacity, Access Control, Room Managed By, and Notify Email.  Otherwise I don't think "capacity" is in the system currently.

Searching by availability is somewhat related to #27 

Having rooms in buildings/floors is related to #15 

I do agree a "Find me a room" would be a nice feature where you could give parameters and it could give you a list of available rooms.  I foresee changing your availability from 8am-9am to 1 hour blocks, kind like doing a "Busy Search."

**Comment by peeroman** [comment]
Yes, I want to search room by resources, capacity and availability. I want to be able to filter room list by required time, capacity, projector ( some rooms have projectors attached to the ceiling and you do not need to get one separately ).
